### ISSUE TITLE
[Suggestion] About oiled foods

### ISSUE THREAD

**Issue by kingkuys2123** [cScotPlay/Rusticated#2]
I would suggest like having "Oiled" as an [Attribute](https://minecraft.fandom.com/wiki/Attribute) for foods than having their own separate items.. This also increases the compatibility for modded foods too, you can do something like read the item if it's a food or something like item tags like **Common** tags "c:foods", it's basically forge tags but fabric

**Comment by cScotPlay** [comment]
Yep, that is the plan.  I haven't been able to implement it the way I have it in my head.  Thanks for the suggestion and for bringing it back on my radar.

**Comment by kingkuys2123** [comment]
Okay! :) take your time, can't wait for the official release!

also May I create a new issue page here that consists a bunch of issues? (Like a bullet list) instead of creating their own issue pages?

**Comment by cScotPlay** [comment]
Yep sure please do.  I'll see what I can do to take them off the list.
